feat(src,fonts,styles): add UTC display, Iosevka font and SynthWave 84 theme

dexxiez opened the pull request:
## Summary
Made some edits to look like my IDE as fully optional customizations all disabled by default. 

## Changes
- Added a new boolean property `ui_show_utc` to control the visibility of UTC time.
- Included UTC time calculation and formatting in the clock functionality.
- Introduced the SynthWave '84 theme to the themes list, adding a new CSS file for the theme.
- Added Iosevka font.
- Edited `README.md` to reflect additions.

dexxiez committed:
feat(src,fonts,styles): add UTC display, Iosevka font and SynthWave '84 theme

- Added a new boolean property `ui_show_utc` to control the visibility of UTC time.
- Included UTC time calculation and formatting in the clock functionality.
- Introduced the SynthWave '84 theme to the themes list, adding a new CSS file for the theme.
- Added Iosevka font

dexxiez committed:
docs(readme): updated readme

unktir commented:
@dexxiez, hi. Thanks for the contribution! I'll see how my free time will be, I hope for understanding

unktir commented on scripts/scripts.js:
```diff
@@ -5,6 +5,8 @@ function Clock() {
     let timeZoneSign = timeZoneOffsetInMinutes > 0 ? '-' : '+';
     let timeZoneOffsetHours = Math.abs(Math.floor(timeZoneOffsetInMinutes / 60));
 
+    let utcStyledString = `"${time.getUTCFullYear()}-${(time.getUTCMonth() + 1).toString().padStart(2, '0')}-${time.getUTCDate().toString().padStart(2, '0')}T${time.getUTCHours().toString().padStart(2, '0')}:${time.getUTCMinutes().toString().padStart(2, '0')}:${time.getUTCSeconds().toString().padStart(2, '0')}.000Z"`;
```
I'm wondering why you wrote this, instead of the using `toISOString()` method of [`Date`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date), which outputs UTC time by default and uses the standard ISO 8601 format (which you trying to get)

dexxiez replied:
Excellent question, I was incredibly hung over and not well. The real reason, was to clean the end bit up (but i never looked at it again). Can be easily fixed with below if you haven't already redone it properly haha 😆 


```js
let xyz = `${time.toISOString().split(".")[0]}.000Z`
```

its fixed in my repo now

unktir committed:
chore(project.json): place the elements in order

unktir committed:
fix(new font): add new font path to the main style file

unktir committed:
fix(new theme): fix the style for the new theme

unktir commented on styles/synthwave-84.css:
I use [Scope inspector](https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#scope-inspector) to get the colors of the code elements in VSCode

unktir commented on styles/main.css:
The new font family was not displayed on the wallpaper, because you forgot to specify the path to it in the main style file

unktir commented:
I hope you will give your comments about my changes and the question about `utcStyledString`

dexxiez committed:
fix(scripts): change the function for getting UTC time

unktir commented on scripts/scripts.js:
```diff
@@ -5,6 +5,8 @@ function Clock() {
     let timeZoneSign = timeZoneOffsetInMinutes > 0 ? '-' : '+';
     let timeZoneOffsetHours = Math.abs(Math.floor(timeZoneOffsetInMinutes / 60));
 
+    let utcStyledString = `${time.toISOString().split(".")[0]}.000Z`;
```
First of all, it wasn't stupidity 😄. And secondly, I think it would be better if we do so `new Date(this.unix * 1000).toISOString();`. I'll make a commit now

unktir committed:
fix(scripts): change the function for getting UTC time

delete utcStyledString and change the function itself. from mutating string to getting string from
unix value

unktir committed:
Merge branch 'main' into feat/dexxiez-edit
